clusterlogs/pipeline.py

```diff
@@ -299,4 +298,6 @@ def process_timings(self):
         if self.timings['group_equals'] != 0 and self.timings['regroup'] != 0:
             self.timings['group_equals'] -= self.timings['regroup']  # group_equals contains regroup
 
-        print(f"Timings:\n{self.timings}")
```

Comment:
I'd like to keep old formatting as it's easier to process by scripts.

Reply:
Sure thing. Should be reverted now